boards/arm/qemu_cortex_m3/qemu_cortex_m3.yaml

```diff
@@ -5,5 +5,7 @@ arch: arm
 toolchain:
   - zephyr
   - gccarmemb
+supported:
+  - slip
```

Comment:
How would this be used in testcases? How can I have a testcase run both for slip, ethernet, wifi if all I care about the is the ip stack as a whole and not the bearer, i.e. I am not testing the interface, I am testing the IP stack.